discord/app_commands/commands.py

```diff
@@ -70,6 +70,11 @@
     # reference the other to prevent type checking errors in callbacks
     from discord.ext.commands import Cog
 
+    ErrorFunc = Callable[
+        [Interaction, AppCommandError],
+        Coroutine[Any, Any, Any],
+    ]
```

Comment:
```suggestion
    ErrorFunc = Callable[[Interaction, AppCommandError], Coroutine[Any, Any, None]]
```

I prefer to keep this in one line. This also satisfies the type checker